src/api-service/__app__/job_templates_manage/__init__.py

```diff
@@ -54,6 +54,9 @@ def delete(req: func.HttpRequest) -> func.HttpResponse:
         return not_ok(request, context="JobTemplateDelete")
 
     entry = JobTemplateIndex.get(request.name)
+    if entry is not None:
+        entry.delete()
```

Comment:
Do have a strategy for integration testing this?

Reply:
Not yet.  I've explored a few options, but will need time to explore first.